Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -208,6 +208,21 @@ static bool hasTransparentContainerStyle(const RenderStyle& style)
             || !(style.borderTopWidth() && style.borderRightWidth() && style.borderBottomWidth() && style.borderLeftWidth()));
 }
 
+static bool usesFillColorWithExtremeLuminance(const RenderStyle& style)
+{
+    auto fillColor = style.fillPaintColor();
```

Comment:
This should use `style().colorResolvingCurrentColor()` instead of the manual current color resolution below.

Reply:
nice, adding `currentcolor` test cases too